prime-router/src/main/kotlin/cli/SettingCommands.kt

```diff
@@ -95,9 +147,7 @@ abstract class SettingCommand(
         payload: String
     ): String {
         val path = formPath(environment, Operation.PUT, settingType, settingName)
-        if (verbose) {
-            echo("PUT $path :: $payload")
-        }
+        verbose("PUT $path :: $payload")
```

Comment:
This `verbose` method is similar to the verbose method in some other our scripts.